### PR TITLE
feat: updated assets page for the EVM account

### PR DESCRIPTION
**Pull request by impelcrypto**
**Pull Request Summary**

* feat: updated assets page for the EVM account

**Check list**

- [ ] contains breaking changes
- [ ] adds new feature
- [x] modifies existing feature (bug fix or improvements)
- [ ] relies on other tasks
- [ ] documentation changes
- [ ] tested on mobile devices

**This pull request makes the following changes:**

**Changes**
* feat: updated assets page for the EVM account
  * combined both XC20 and ERC20 assets on the same panel
  * put native token on the top
* fix: hid XVM assets panel because AA might replace it
  
=Before=
image

=After=

image

Search `USD`
image

### CODE REVIEW

**Review by gluneau — APPROVED** [review]
looks good, new tests passes locally

**Comment by impelcrypto** [comment]
@Kahonnohak 

> Would you like to merge it? or wait until Unified account?

I'll merge it today.

> If you are merging please add the same border line with what we have in Astar Native.

Updated.
image

File: src/components/assets/EvmNativeToken.vue [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+    const { nativeTokenUsd } = usePrice();
+    const store = useStore();
+    const isH160 = computed<boolean>(() => store.getters['general/isH160Formatted']);
+    const isLoading = computed<boolean>(() => store.getters['general/isLoading']);
+
+    const nativeTokenImg = computed<string>(() =>
+      getTokenImage({ isNativeToken: true, symbol: nativeTokenSymbol.value })
+    );
+
+    const updateStates = async (nativeTokenUsd: number): Promise<void> => {
```
Comment (bobo-k2) [inline src/components/assets/EvmNativeToken.vue:94]: I believe we should not have business or data logic in components. You don't need to do refactor now in this PR, but have in mind for the future components. I will try to remind myself also

**Review by bobo-k2 — APPROVED** [review]
LGTM

**Review by Kahonnohak — APPROVED** [review]
LGTM

**Pull request merged**